build.gradle

```
@@ -879,7 +884,8 @@ void fetchExternalTools(String configName, List packages, File destdir, boolean
                     if (IS_LINUX || IS_MAC) {
                         // use native tar to support symlinks
                         pkgdir.mkdirs()
-                        exec {
+                        def execOps = project.services.get(ExecOperations)
+                        execOps.exec { ExecSpec spec ->
```

```suggestion
                        execOps.exec { spec ->
```

We generally don't include the type of a closure argument, unless it's ambiguous (which I don't think it is here).

---

Thank you, removed the type from all such instances.